quickwit-cluster/src/lib.rs

```diff
@@ -26,7 +26,7 @@ use std::sync::Arc;
 use anyhow::bail;
 use chitchat::transport::UdpTransport;
 use chitchat::FailureDetectorConfig;
-use quickwit_config::QuickwitConfig;
+use quickwit_config::QuickwitConfigObject;
```

Comment:
Since `QuickwitConfigObject` is the object that developpers are mostly going to manipulate`, I'd rather name it `QuickwitConfig` and call the intermediate object `QuickwitConfigBuilder` or something like that.